src/Hooks/dllmain.cpp

```diff
@@ -0,0 +1,11 @@
+#include <Windows.h>
```

Comment:
Use precompiled header pch.h for external includes

Reply:
You can look how Mateush done this

Reply:
done